src/connector/Cargo.toml

```diff
@@ -125,7 +119,11 @@ rustls-native-certs = "0.7"
 rustls-pemfile = "2"
 rustls-pki-types = "1"
 rw_futures_util = { workspace = true }
-sea-schema = { version = "0.14", features = ["default", "sqlx-postgres", "sqlx-mysql"] }
+sea-schema = { version = "0.14", default-features = false, features = [
+    "discovery",
+    "sqlx-postgres",
+    "sqlx-mysql",
+] }
```

Comment:
This is changed because I found `cargo check -p risingwave_connector` compile failed.